### PR TITLE
Add Simon to contributors

### LINKED ISSUE #24: Add SimonNord to Contributors
1. Clone the repo (help: https://help.github.com/en/github/creating-cloning-and-archiving-repositories/cloning-a-repository)
2. Open the cloned repo with your code editor (aka IDE)
3. Create a new branch (command: git branch name-of-your-branch)
3. go to the file package.json
4. Add yourself to the contributors field 
5. Push the branch (git push --set-upstream origin name-of-your-branch)
6. Open a Pull Request (help: https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/about-pull-requests)

The flow above is with the git cli. But if you feel more comfortable with the GitHub app (https://desktop.github.com/), that is fine as well. 

The important part is to add yourself to contributors and open a PR. The end result should look like this: https://github.com/HandsOnLearning/the-project/pull/1

If you need help, ask here, failing is learning!

### PR DESCRIPTION
**Pull request by SimonNord**
fixes #24

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -6,6 +6,7 @@
   "author": "Mo Sattler",
   "contributors": [
     "Mo Sattler",
+    "Simon Nordström,
```
Comment (MoSattler) [inline package.json:9]: You forgot to close the string with a `"`

**Comment by SimonNord** [comment]
Oh is it failing since I didnt have the latest update of Node when I commited?

**Comment by MoSattler** [comment]
@SimonNord no, it's that you forgot to close a string :)

File: package.json (hunk 1 above)
Comment (SimonNord) [reply package.json:9]: Yeah one backspace top much

**Pull request merged**